### DISCUSSION TITLE
Type error: Cannot find type definition file for 'e2e'.

### DISCUSSION THREAD

**Discussion by es-ki** [vercel/next.js#37457] [Help] [answered]
We deployed an application created in Next.js to Vercel from the CLI.  
When we checked the Deployment Status of Vercel, the following error was displayed.  
![err](https://user-images.githubusercontent.com/48976713/172033224-4a0a6632-76c6-4abe-ae28-4bdb7ce4468d.JPG)
```
info  - Checking validity of types...
Failed to compile.
Type error: Cannot find type definition file for 'e2e'.
The file is in the program because:
Entry point for implicit type library 'e2e'
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
Error: Command "yarn run build" exited with 1
```

It says that the build process failed, so I tried to build locally and got the same error message.  
![cli_err](https://user-images.githubusercontent.com/48976713/172033458-c8c67482-25f1-4d72-b6fb-c1acae7325a6.jpg)

Could you please let me know the cause of this error and if there is a solution?  
I searched for the error text but could not find any reports of the same phenomenon or a solution.

**Comment by icyJoseph** [comment, marked as answer]
Ah, oh boy, this is a TypeScript error... what does your `package.json` and `tsconfig.json` look like? IIRC, there must be a package that's called `@types/` but has no types, or something like that... Hardly Next.js related, but I can try to help. Do you have a link to a repository?

**Reply by es-ki** [reply]
Thanks for the help!  
It seems that the problem was that the ```devDependencies``` in ```package.json``` contained a strange type "@types/", which I removed and now it works fine.
